files/blink.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.syco.blink</id>
```

Comment:
I don't think this app-id is correct. this isn't the canonical location for this app.

Reply:
It's not the canonical location, but the developers said many times that contributors are more than welcome.
However, I didn't want to use "com.agprojects.blink" in case they want to push an official build in the future.

Reply:
That's not how things work for flathub. You should use the 'official' app-id (and ideally discuss it with the app maintainers, because getting the 'wrong' app-id is a massive pain), and if the official developers want to take over then they'd be added to the app you started and take over from there - that way we don't end up with lots of different apps that are in fact entirely the same, splintering fixes and updates and looking very confusing. It's for the same reason that we need people to at least try to push the changes they make (like desktop files and appdata) to upstream.

Of course, if you wanted to entirely fork the app and make changes that way as a new upstream then that would be a different conversation.

Reply:
I understand, I change the app id to com.icanblink.blink since the official website is http://icanblink.com/.
Unfortunately the developer are being very unhelpful borderline rude. On the mailing list they made very clear they are not interested in supporting anything other than debian/ubuntu.
This app itself uses python2 (now end of life) and many other deprecated libraries with no plans to updating them. Arch Linux already removed some of them from the repo making it impossible to easily install it.
That's why I thought flatpak could solve the library issue and allow people to keep using the app. It's a good app, shame it's maintained so poorly.
Also, I prefer to stay as far as possible from python, so I can't fork and maintain it. However I'm happy to package it and make it available.